cmd/antrea-agent/agent.go

```diff
@@ -203,7 +203,7 @@ func run(o *Options) error {
 		o.config.HostProcPathPrefix,
 		nodeConfig,
 		k8sClient,
-		podUpdates,
+		entityUpdates,
```

Comment:
Could we pass a nil in the AntreaAgent case, as we will never have ExtEntity updates?

Reply:
I guess it means to replace the podUpdate channel, not add a new channel for external entity.
Question for this change: the channel was designed to speed up networkpolicy enforcement as containers can start very fast and may communiate with others before agent gets Pod events from K8s API. Does the externalEntity scenario need this and use the channel?

Reply:
@tnqn In VM agent use case, the events to NetworkPolicyController is uncorrelated, That is,  it may first receives internal AddrGrp/AppliedTo referencing to Pod/ExternalEntity from K8s client,  then receives Pod/External update; or vice versa. Hope this answers ur question. 

Or maybe there is a good reason that we should not mingle ExternalEntity and Pod update channel at all?

Reply:
I mistook we will have two channels. Then the current change is fine to me.